src/BuildToolsUnitTests/AOT/AOTSchemaTests.cs

```diff
@@ -144,7 +146,7 @@ public void ParseProtoToModel(string protoPath)
 
         json = JsonConvert.SerializeObject(parsedFromCode, JsonSettings);
         _output.WriteLine(json);
-        jsonPath = Path.ChangeExtension(protoPath, ".pjson");
+        jsonPath = Path.ChangeExtension(protoPath, ".cs.json");
```

Comment:
only thing to watch here is whether the csproj wildcard and nesting works nicely; that was the main reason I went for a single-part extension (IIRC it wasn't happy initially)

Reply:
changed that to `pjson` back!